tap/go.mod

```diff
@@ -3,15 +3,14 @@ module github.com/up9inc/mizu/tap
 go 1.16
 
 require (
-	github.com/bradleyfalzon/tlsx v0.0.0-20170624122154-28fd0e59bac4 // indirect
+	github.com/bradleyfalzon/tlsx v0.0.0-20170624122154-28fd0e59bac4
 	github.com/google/gopacket v1.1.19
-	github.com/romana/rlog v0.0.0-20171115192701-f018bc92e7d7
+	github.com/up9inc/mizu/shared v0.0.0
 	github.com/up9inc/mizu/tap/api v0.0.0
-	golang.org/x/net v0.0.0-20210224082022-3d97a244fca7
-	golang.org/x/sys v0.0.0-20210225134936-a50acf3fe073
-	golang.org/x/term v0.0.0-20210220032956-6a3ed077a48d
-	golang.org/x/text v0.3.5
-	golang.org/x/tools v0.0.0-20210106214847-113979e3529a
+	golang.org/x/net v0.0.0-20210224082022-3d97a244fca7 // indirect
```

Comment:
indirect?

Reply:
@RoyUP9 why do you type "indirect?" whenever you see `// indirect`? Could you please share some context?

If the driver program and a Go plugin has a shared dependency, they should have the exact same version of that dependency otherwise the plugin cannot be loaded. (throws a segmentation fault) Standard library is outside of this since it comes with the language's itself.

Reply:
what i meant is that i'm not sure if we want the indirect to be there or not (because it wasn't there before the change).
to be honest, i don't really know when it should be there and when it shouldn't.